agent/www/index.html

```diff
@@ -51,6 +51,37 @@
     <script src="components/connections/connections_ctrl.js"></script>
   </head>
   <body>
+
+  <script type="text/ng-template" id="/peer_lost.html">
+    <div class="message-dialog-pf">
+      <div class="modal-header">
+        <h4 class="modal-title" id="myErrorDialogLabel">Server Connection Lost</h4>
+      </div>
+      <div class="modal-body">
+        <div>
+          <p class="lead pficon pficon-error-circle-o">
+            Server Connection Lost
+          </p>
+          <p>
+            This may mean that a server component is being restarted. If this is the case, the console
+            will reconnect automatically. <!--Alternatively, the address space to which the console was connected
```

Comment:
Perhaps 'This may mean that a server component is being restarted or deleted. If it is restarted, the console will reconnect automatically.' ?

Reply:
I don't think server component being delete makes sense to the user.
How about?

```
This may mean that the address space to which the console is connected has been permanently deleted, or it
could be indicative of a transient issue such as a network problem or server component restart.  If the
issue is transient, the console will reconnect automatically once the problem is resolved.
```